chrome/manifest.json

```diff
@@ -43,7 +43,15 @@
       "suggested_key": {
         "default": "Ctrl+Shift+K",
         "mac": "Command+Shift+K"
-      }
+      },
+      "description": "Open the Kagi extension"
+    },
+    "summarize-active-page": {
+      "suggested_key": {
+        "default": "Ctrl+Shift+L",
+        "mac": "Command+Shift+L"
+      },
+      "description": "Summarize the currently active page"
```

Comment:
New shortcut to instantly summarize the page! For Chrome I chose the letter next to K.